site/content/entry/yuzu-progress-report-nov-2021/index.md

```diff
@@ -0,0 +1,330 @@
+[v1993](https://github.com/v1993) got a report that yuzu’s window will flicker if the user holds the Esc key when trying to exit fullscreen. 
+[That won’t happen any more](https://github.com/yuzu-emu/yuzu/pull/7353).
+
+german77 discovered that holding a hotkey would constantly spam it, that made him angry, and [this behaviour is no more](https://github.com/yuzu-emu/yuzu/pull/7355).
+
+[heinarmann](https://github.com/heinermann) [replaced the emulation shortcuts and states](https://github.com/yuzu-emu/yuzu/pull/7419):
+
+`Start` has been removed and `Continue` was moved to `Pause` when applicable.
+Exiting the emulator will now work while in fullscreen.
+`Restart` will no longer cause a crash.
+`Load Amiibo` was improved. We still have issues with several games when using this function, so consider it in a beta state.
+`TAS` [got it’s own submenu](https://github.com/yuzu-emu/yuzu/pull/7406) in `Tools > TAS`.
+
+{{< single-title-imgs
+    " "
+    "./tasbug.png"
+    "./tasfix.png"
+    >}}
+
+## Kernel fixes
+
+Morph fixed [a compilation warning](https://github.com/yuzu-emu/yuzu/pull/7278) in the kernel `SVC::WaitSynchronization`, where the parameter "num_handles" was wrongly 
+declared as unsigned integer when it was actually a signed integer.
+
+heinermann discovered a bug in the kernel that crashed yuzu upon exiting.
+
+This happened because dummy threads in the kernel weren't getting destroyed properly. When you exited yuzu during emulation or after stopping emulation, these threads 
+stayed alive even though the kernel got destroyed, thus causing yuzu to crash.
+
+[By making the kernel own these threads](https://github.com/yuzu-emu/yuzu/pull/7359), we ensure they get destroyed when the kernel gets destroyed. 
+Thus the crash is mitigated.
+
+[OatmealDome](https://github.com/OatmealDome) fixed a bug that caused some homebrew to not boot. The fix was to 
+[pass default threadinfo when the kernel creates a thread](https://github.com/yuzu-emu/yuzu/pull/7320), thus allowing homebrew to boot.
+
+## Skyline framework, part 1
+
+[itsmeft24](https://github.com/itsmeft24), [jam1garner](https://github.com/jam1garner), and Morph started working in adding support for the 
+[Skyline](https://github.com/skyline-dev/skyline) modding framework. 
+This will allow for more intricate code mods, such as using [ARCropolis](https://github.com/Raytwo/ARCropolis), a mod loader for Smash Ultimate made by 
+[Raytwo](https://github.com/Raytwo) and [blujay](https://github.com/blu-dev).
+
+[All](https://github.com/yuzu-emu/yuzu/pull/7393) [these](https://github.com/yuzu-emu/yuzu/pull/7394) [PRs](https://github.com/yuzu-emu/yuzu/pull/7407) are part of the 
+on-going effort to implement support for modding framework in yuzu.
+
+There are more juicy additions, but they didn't make it in time to be mentioned in this article, we will continue to expand on Skyline's progress in December's report.
+See the current progress [here](https://github.com/yuzu-emu/yuzu/issues/7392).
```

Comment:
```suggestion
[itsmeft24](https://github.com/itsmeft24), [jam1garner](https://github.com/jam1garner), and Morph started working on adding support for the 
[Skyline](https://github.com/skyline-dev/skyline) modding framework. 
This will allow for more intricate code mods, such as using [ARCropolis](https://github.com/Raytwo/ARCropolis), a mod loader for Smash Ultimate made by 
[Raytwo](https://github.com/Raytwo) and [blujay](https://github.com/blu-dev).

All these PRs ([[1](https://github.com/yuzu-emu/yuzu/pull/7393)], [[2](https://github.com/yuzu-emu/yuzu/pull/7394)], [[3](https://github.com/yuzu-emu/yuzu/pull/7407)) are part of the 
on-going effort to implement support for the modding framework in yuzu.

There are more juicy additions, but they didn't make it in time to be mentioned in this article, we will continue to expand on Skyline's progress in December's report.
See the current progress [here](https://github.com/yuzu-emu/yuzu/issues/7392).
```